### PR TITLE
Reset refreshing_repos flag for stuck users

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: lib/tasks/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace :user do
+  desc "Reset refreshing_repos flag for stuck users"
+  task reset_refreshing_repos_for_stuck_users: :environment do
+    stuck_users = User.where(
+      "refreshing_repos = true AND updated_at < NOW() - interval '10 minutes'"
```
[the remark this answers is not in the document]
Comment (gylaz) [reply lib/tasks/user.rb]: > NOW() - interval '10 minutes'

Neat!

File: lib/tasks/user.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    )
+
+    stuck_users.update_all(refreshing_repos: false)
```
[the remark this answers is not in the document]
Comment (gylaz) [reply lib/tasks/user.rb]: I guess the alternative here would be iterating and calling `update`. Should we disable this rule?

[the remark this answers is not in the document]
Comment (salbertson) [reply lib/tasks/user.rb]: I like that Hound pointed this out, but let's ignore it in this specific case.

File: lib/tasks/user.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (salbertson) [reply lib/tasks/user.rb]: Hopefully there's no good reason for repos to take longer than ten minutes to reload.

**Comment by salbertson** [comment]
@gylaz I'm thinking we can run this on some interval, let me know what you think.

**Pull request merged**